Doc/library/signal.rst

```diff
@@ -157,6 +157,8 @@ The variables defined in the :mod:`signal` module are:
 
    Alias to :data:`SIGCHLD`.
 
+   .. availability:: not macOS
```

Comment:
```suggestion
   .. availability:: not macOS.
```

I think it would be better to add periods to make it consistent with the rest of the file.